doc/source/changes/version_0_26.rst.inc

```diff
@@ -6,12 +6,15 @@
   results, but inside a function local_arrays will return only arrays local to the function, while global_arrays will
   return only arrays defined globally.
 
+* indicator `*` is appended to the title window when unsaved changes are detected in viewer (closes :editor_issue:`21`).
```

Comment:
* replace "indicator `*`" by either "a `*` indicator" or even "a `*` symbol" (my personal preference)
* s/title window/window title/
* s/in viewer/in the viewer/